azure-spring-boot-samples-6.x-pom.xml

```diff
@@ -57,4 +57,25 @@
     </plugins>
   </build>
 
+  <repositories>
+    <repository>
+      <id>spring-milestones</id>
+      <name>Spring Milestones</name>
+      <url>https://repo.spring.io/milestone</url>
```

Comment:
Why need this?

Reply:
Because the stable version 6.0.0-beta.4 is needed, this version is based on a Spring milestone version.